src/run/gtimes/_rt.ts

```diff
@@ -0,0 +1,43 @@
+import Discord from "discord.js";
+
+export function _rt(client: Discord.Client) {
+  client.on("messageCreate", async (message) => {
+    if (message.author.bot) return;
+    if (message.content.startsWith("RT:")) {
+      const regex = /https:\/\/discord.com\/channels\/\d+\/(\d+)\/(\d+)/;
+      const str = message.content;
+      const match = str.match(regex);
+      if (match === null) return;
+      const channelID = match[1];
+      const messageID = match[2];
+      const quoteChannel = client.channels.cache.get(`${channelID}`);
```

Comment:
```suggestion
      const quoteChannel = client.channels.cache.get(channelID);
```
最初から `string` のモノをテンプレート文字列のプレースホルダーにかけている箇所が多すぎる